packages/typespec-go/src/tcgcadapter/clients.ts

```diff
@@ -263,13 +265,15 @@ export class clientAdapter {
       }
     }
 
-    this.adaptMethodParameters(sdkMethod, method);
+    this.adaptMethodParameters(sdkMethod, method, paramMapping);
```

Comment:
It looks like `adaptMethodParameters` is populating `paramMapping` yes? If so, can we change this so `adaptMethodParameters` returns it instead of using an out param?

Reply:
changed.